src/main/java/frc/robot/Constants.java

```diff
@@ -101,6 +101,10 @@
   public static final double TARGET_HEIGHT = 60.0;
   public static final double LIMELIGHT_HEIGHT = 39.5;
 
+  //#endregion
+
+  //#region Turret Details
```

Comment:
We now have two "Turret Details" regions. That doesn't seem right.

Reply:
resolved in 3d8d1f5